packages/core-ethereum/package.json

```diff
@@ -9,7 +9,8 @@
   "main": "lib/index.js",
   "scripts": {
     "clean": "rimraf ./lib ./tsconfig.tsbuildinfo",
-    "test": "mocha --parallel",
+    "coverage": "nyc --reporter=html mocha",
```

Comment:
Ouf of scope: Should we add a coverage step to our CI and upload the html report as an artifact, or maybe use a different reporter in CI?

Reply:
Yeah, but this is indeed OOS for this PR. I suggest we follow-up on that on next sprint or so.